cmd/go-swagger/swagger.yml

```diff
@@ -342,12 +342,12 @@ paths:
             description: The response when succesful query happens
         default:
           description: Error occured
-  /xss/Sink/buffered-query/{safety}:
+  /xss/reflectedXss/buffered-query/{safety}:
```

Comment:
I think if you rename these to `reflectedXSS` then there will be no need to add `Xss` to the list of initialisms in `regen.go`